xml/obs/openSUSE:Slowroll:Staging-images.xml

```diff
@@ -0,0 +1,10 @@
+<openQA
+    project_pattern="openSUSE:Slowroll:Staging"
+    dist_path="images"
```

Comment:
i think this has to be `images/x86_64`. bouble check because i am not sure

Reply:
updated.